framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -535,7 +534,9 @@ private static String checkGetParam(HttpServletRequest request, String key) thro
       if (StringUtils.isBlank(contentType)) {
         return null;
       }
-      if (APPLICATION_JSON.toLowerCase().contains(contentType)) {
+      if (contentType.contains(MimeTypes.Type.APPLICATION_JSON.asString())
```

Comment:
if `contentType.contains(MimeTypes.Type.APPLICATION_JSON_UTF_8.asString()) == ture`
then  `contentType.contains(MimeTypes.Type.APPLICATION_JSON.asString() must be true.

Reply:
The `content-type` will be true if the condition is met.

Reply:
I mean the last two rulings are not necessary,  if you insist on doing this, all the scenes need to be added